src/service/CredentialService.ts

```diff
@@ -30,7 +30,7 @@ export class CredentialService {
   public static async set(auth: any, type_id: string | null, access_key: string, credential: any | null) {
     const CredentialRepository = new Repository().getCredentialRepository()
     type_id = await _verify(auth, ["self", "parent"], type_id)
-    if (credential === null) {
+    if (credential !== null) {
```

Comment:
Can you keep the statement as `=== null` and swap the contents of the if and else?

Reply:
ok